src/config/configure.in

```diff
@@ -2552,6 +2591,30 @@ fi
 if test "x$hypre_using_umpire" = "xyes"
 then
    AC_DEFINE(HYPRE_USING_UMPIRE, 1, [Define to 1 if using UMPIRE])
+   if test "x$HYPRE_UMPIRE_RPATH" != "x"
+   then
+      LDFLAGS="${LDFLAGS} ${HYPRE_UMPIRE_RPATH}"
+      AC_MSG_NOTICE([Adding Umpire rpath to LDFLAGS: ${HYPRE_UMPIRE_RPATH}])
+   fi
+   if test "$hypre_user_gave_umpire_inc" != "yes" || \
+      test "$hypre_user_gave_umpire_lib" != "yes" && test "$hypre_user_gave_umpire_libs" != "yes" || \
+      test "$hypre_user_gave_umpire_lib_dirs" != "yes"
+   then
+      AC_MSG_ERROR([
+===============================================================
+Umpire has been enabled for GPU builds to improve performance.
+However, Umpire include and/or library paths were not provided.
+It is strongly recommended to specify Umpire locations to avoid
```

Comment:
Is this "strongly recommended" or simply required. I would just delete this and let the next statement be "Provide the following..." It's not necessary to mention performance twice in the comment anyway.

Reply:
Fixed in [d329ac1](https://github.com/hypre-space/hypre/pull/1360/commits/d329ac1f85bbcb1a420ba9ccc62d3d766ecd83ca)